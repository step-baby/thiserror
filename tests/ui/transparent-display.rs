use thiserror::EnumDisplay;

#[derive(EnumDisplay, Debug)]
#[display(transparent)]
#[display("...")]
pub struct Error(anyhow::Error);

fn main() {}
