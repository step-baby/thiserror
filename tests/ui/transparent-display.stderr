error: cannot have both #[display(transparent)] and a display attribute
 --> tests/ui/transparent-display.rs:5:1
  |
5 | #[display("...")]
  | ^^^^^^^^^^^^^^^
